<launch>

    <node pkg="zeabus_elec_ros" type="power_distributor" name="zeabus_elec_power_dist" respawn="true">
        <param name="i_initial_io_direction" value="65535"/>
        <param name="i_initial_io_pin_state" value="7936"/>
    </node>

</launch>
